contrib/aks/setup_install_ho_on_aks.sh

```diff
@@ -2,50 +2,9 @@
 set -x
 
 # Constants
-LOCATION="eastus"
-RG="hc-test"
-DNS_ZONE_NAME="azure.blah.com"
-EXTERNAL_DNS_NEW_SP_NAME="ExternalDnsServicePrincipal"
-CLUSTER_NAME=<cluster_name>
-AZURE_CREDS=<path_to>/credentials
-AZURE_BASE_DOMAIN=<domain>
-PULL_SECRET=<path_to>/all-the-pull-secrets.json
-PATH_TO_HYPERSHIFT_BIN=<path>
-RELEASE_IMAGE=quay.io/openshift-release-dev/ocp-release-nightly@sha256:b619707647800f7c382e7cb36e7b1026d82a576661274baffaf0585dd257fd1d
-PATH_TO_AZURE_JSON=<path>
-
-######################################## ExternalDNS Setup ########################################
-# Clear out existing Azure RG
-az group delete -n ${RG} --yes
-
-# Create Azure RG and DNS Zone
-az group create --name ${RG} --location ${LOCATION}
-az network dns zone create --resource-group ${RG} --name ${DNS_ZONE_NAME}
-
-# Creating a service principal
-DNS_SP=$(az ad sp create-for-rbac --name ${EXTERNAL_DNS_NEW_SP_NAME})
-EXTERNAL_DNS_SP_APP_ID=$(echo "$DNS_SP" | jq -r '.appId')
-EXTERNAL_DNS_SP_PASSWORD=$(echo "$DNS_SP" | jq -r '.password')
-
-# Assign the rights for the service principal
-DNS_ID=$(az network dns zone show --name ${DNS_ZONE_NAME} --resource-group ${RG} --query "id" --output tsv)
-az role assignment create --role "Reader" --assignee "${EXTERNAL_DNS_SP_APP_ID}" --scope "${DNS_ID}"
-az role assignment create --role "Contributor" --assignee "${EXTERNAL_DNS_SP_APP_ID}" --scope "${DNS_ID}"
-
-# Creating a configuration file for our service principal
-cat <<-EOF > ${PATH_TO_AZURE_JSON}/azure.json
```

Comment:
Can the setup of the external dns be found elsewhere?

Reply:
Yeah it's already in its own file in this same directory.